### ISSUE TITLE
Outdated plugin: owasp dependency-check (3.1.2 instead of 3.1.1)

### ISSUE THREAD

**Issue by dzc34** [Asqatasun/Asqatasun#217]
## Observed behaviour
outdated dependency: **owasp dependency-check** plugin

## Expected behaviour
-  upgrade owasp dependency-check plugin    (3.1.2 instead of 3.1.1)
   - https://jeremylong.github.io/DependencyCheck/dependency-check-maven/
   - https://github.com/jeremylong/DependencyCheck
   - release notes
     - https://github.com/jeremylong/DependencyCheck/blob/master/RELEASE_NOTES.md
     - https://groups.google.com/forum/#!forum/dependency-check

## File
[pom.xml](https://github.com/Asqatasun/Asqatasun/blob/develop/pom.xml#L93)

### How to test it?
```bash
# pre-requisite: 
# - git, maven
# - fork  the repository on Github
git clone https://github.com/<user>/Asqatasun.git
     # instead of:   git clone https://github.com/Asqatasun/Asqatasun.git
cd Asqatasun
git checkout develop
git checkout -b fix217

mvn dependency-check:aggregate
    # see ./target/dependency-check-report.html

# commit / push  + PR (pull request on Github)
git add .
git commit -m "Fixed #217  - Upgraded owasp:dependency-check (3.1.2 instead of 3.1.1) "
git push origin fix217
```

**Comment by Tyraelium** [comment]
hi, thanks for the issue I try to do it.

**Comment by mfaure** [comment]
Fixed by #226
